### DISCUSSION TITLE
Opening the Personalization settings via the context menu SOMETIMES closes that settings window.

### DISCUSSION THREAD

**Discussion by Hybrid-Divide** [valinet/ExplorerPatcher#3592] [Q&A] [answered]
When I select "Personalization" via the context menu on the desktop, shortly after the settings are opened up, it closes.  Doesn't happen everytime.  But it's more than once.

Here's the log via the Event Viewer.
---------------------------------------
Faulting application name: SystemSettings.exe, version: 10.0.22621.3958, time stamp: 0x052f4222
Faulting module name: ExplorerPatcher.amd64.dll, version: 22621.3527.65.5, time stamp: 0x6644866d
Exception code: 0xc0000005
Fault offset: 0x000000000000d5f3
Faulting process id: 0x0x2D20
Faulting application start time: 0x0x1DAF7994585D99A
Faulting application path: C:\Windows\ImmersiveControlPanel\SystemSettings.exe
Faulting module path: C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll
Report Id: f37dc686-dd93-4f3b-a660-93387cf3985d
Faulting package full name: windows.immersivecontrolpanel_10.0.6.1000_neutral_neutral_cw5n1h2txyewy
Faulting package-relative application ID: microsoft.windows.immersivecontrolpanel

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.  Please include if you are running x86-64 or Arm.  Is it windows 11 22621.3958?
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

**Reply by Hybrid-Divide** [reply]
Windows version as follows:
![image](https://github.com/user-attachments/assets/ac58f760-bc8f-419d-b9a7-55b772fe7c05)

CPU Type: X86-64 (AMD Ryzen 7 5800X3D
ExplorerPatcher is the latest version (See screenshots below)
The only other software I'm using that modifies the UI is OpenShell for the start menu.
(But notice that the Event Viewer logs from the original post flags ExplorerPatcher with the settings error.)

![image](https://github.com/user-attachments/assets/4d19282b-e648-4193-bc50-a6bab1d4f71e)
![image](https://github.com/user-attachments/assets/0deff675-d645-4293-a6e6-8f2b654bedd1)
![image](https://github.com/user-attachments/assets/da363a70-e5b1-4aa9-b4c6-07d324470d6d)
![image](https://github.com/user-attachments/assets/6b57ab93-77fa-47d0-a202-22d0865c846e)
![image](https://github.com/user-attachments/assets/297d2280-00b1-4950-bf4a-d3b5bac4e4d7)
![image](https://github.com/user-attachments/assets/0e262bc7-8fc2-4f51-883f-0226e782013f)
![image](https://github.com/user-attachments/assets/b12dfdb5-89bf-44f3-88f0-15e795ebfa59)

Hope this helps.

**Comment by pyrates999** [comment, marked as answer]
You enabled the option register as shell extension, disable it by opening up the EP properties application, go to **File Explorer**, and untick **Register as shell extension**.

Here's why: https://github.com/valinet/ExplorerPatcher/wiki/Using-ExplorerPatcher-as-shell-extension

**Reply by Hybrid-Divide** [reply]
Interesting!  Done.
Thank you!

**Reply by pyrates999** [reply]
Please close this.  And check what an option does before you enable it if you're not sure.

**Reply by Hybrid-Divide** [reply]
Right.  Will do. Thanks.
